pyproject.toml

```diff
@@ -61,6 +61,7 @@ dependencies = [
     "pillow>=9.4.0",
     "safer>=4.8.0",
     "semver>=3",
+    "requests-file>=1.5.1",
```

Comment:
To me, this is only a test depdendency, and so I would only include it in the tests group.